### ISSUE TITLE
installation with mactex2018 not working

### ISSUE THREAD

**Issue by aaltland** [ketan/BeautifyLatex#19]
I managed to get the package to work with mactex2016 following the below installation instructions. However, after upgrading to 2018 and following the same procedure, I get

Cannot write to 'indent.log': Permission denied at /Library/Perl/5.18/Log/Dispatch/File.pm line 100

Somehow latexindent (working after several packages have been installed) attempts to write to a directory with protected permissions. 

Any idea how to fix this?

**Comment by drmattprior** [comment]
I have the same problem. Have you had any success fixing it?

**Comment by aaltland** [comment]
Nope. I went as far as to re-install mactex & try from scratch. Same result. The latexindent documentation is far too complicated for me to understand. Gave up on it.
